src/pages/faretype.tsx

```diff
@@ -1,79 +1,106 @@
-                    <fieldset className="govuk-fieldset" aria-describedby="faretype-page-heading">
-                        <legend className="govuk-fieldset__legend govuk-fieldset__legend--xl">
-                            <h1 className="govuk-fieldset__heading" id="faretype-page-heading">
-                                What type of fare would you like to provide?
-                            </h1>
-                        </legend>
-                        <div className="govuk-radios">
-                            <div className="govuk-radios__item">
-                                <input
-                                    className="govuk-radios__input"
-                                    id="faretype-single"
-                                    name="faretype"
-                                    type="radio"
-                                    value="single"
-                                />
-                                <label className="govuk-label govuk-radios__label" htmlFor="faretype-single">
-                                    Single - Point to Point
-                                </label>
-                            </div>
-                            <div className="govuk-radios__item">
-                                <input
-                                    className="govuk-radios__input"
-                                    id="faretype-period"
-                                    name="faretype"
-                                    type="radio"
-                                    value="period"
-                                />
-                                <label className="govuk-label govuk-radios__label" htmlFor="faretype-period">
-                                    Period Tickets
-                                </label>
-                            </div>
-                            <div className="govuk-radios__item">
-                                <input
-                                    className="govuk-radios__input"
-                                    id="faretype-return"
-                                    name="faretype"
-                                    type="radio"
-                                    value="return"
-                                    disabled
-                                    aria-disabled="true"
-                                />
-                                <label className="govuk-label govuk-radios__label" htmlFor="faretype-return">
-                                    Return
-                                </label>
+type FareTypeProps = {
+    errors: ErrorInfo[];
+};
+
+const FareType = ({ errors = [] }: FareTypeProps): ReactElement => {
+    return (
+        <Layout title={title} description={description}>
+            <main className="govuk-main-wrapper app-main-class" id="main-content" role="main">
+                <form action="/api/fareType" method="post">
+                    <ErrorSummary errorHref="#radio-buttons" errors={errors} />
```

Comment:
Can the errorHref redirect to an ID other than the `#radio-buttons`? If so, it might look better with redirecting to `#page-heading` just so the user has a signpost to see where they are on the page - what do you think?  🤔

Reply:
Unsure what you mean here - the only error a user can make on this page is not selecting a radio button, so we know the Href will always be on the radio button form. Giving the ErrorSummary component the radio button's Href will mean the error message always links them to the form where the error was made.

Reply:
I was thinking back to when we had a chat about how the pages looked when the href forces the user down the page to the buttons. I had a play around with the IDs available and thought we could replace `errorHref="#radio-buttons"` with `errorHref="#page-heading"` so that the page heading is within sight when the user is directed down the page - just to give a little more context to them. But I wasn't sure if that href serves any other purposes?

Reply:
Im not sure if we're getting wires crossed here. The tabIndex = {-1} is what sets what the user's focus is on when they are redirected, and the href only sets what their focus is brought to when they click on the error message.